[LLM] Quickstart and how-tos (#2845)
scaleway/docs-content: ai-data/llm-inference/how-to/monitor-deployment.mdx

---
meta:
  title: How to monitor a LLM Inference deployment
  description: This page explains how to monitore a LLM Inference deployment
content:
  h1: How to monitor a LLM Inference deployment
  paragraph: This page explains how to monitore a LLM Inference deployment
tags: ai monitoring
dates:
  validation: 2024-03-06
  posted: 2024-03-06
categories:
  - ai-data
---

Monitoring allows you to monitor your deployment.

> <Macro id="iam-requirements" />

Reviewer: ```suggestion
<Macro id="requirements" />
```

> <Message type="requirement">

Reviewer: ```suggestion
```

>   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)

Reviewer: ```suggestion
  - A Scaleway account logged into the [console](https://console.scaleway.com)
```

  - You have a [LLM Inference deployment](/ai-data/llm-inference/quickstart/)

>   - You have [enabled your Cockpit](/observability/cockpit/how-to/activate-cockpit/)

Reviewer: ```suggestion
  - An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```

> </Message>

Reviewer: ```suggestion
```

## How to monitor your LLM dashboard

1. Click **LLM Inference** in the **AI & Data** section of the [Scaleway console](https://console.scaleway.com) side menu. A list of your deployments displays.

> 2. Click a deployment name or <Icon name="more" /> > **More info** to access the deployment dasdhboard.

Reviewer: ```suggestion
2. Click a deployment name or <Icon name="more" /> > **More info** to access the deployment dashboard.
```
